recipes/openh264/all/conanfile.py

```diff
@@ -1,8 +1,29 @@
-from conan.tools.microsoft import msvc_runtime_flag
-from conans import ConanFile, tools, AutoToolsBuildEnvironment
+from conan import ConanFile
+from conan.tools.env import VirtualBuildEnv
+from conan.tools.files import apply_conandata_patches, copy, export_conandata_patches, get, rmdir, replace_in_file, chdir
+from conan.tools.gnu import Autotools, AutotoolsToolchain
+from conan.tools.apple import is_apple_os, fix_apple_shared_install_name
+from conan.tools.layout import basic_layout
+from conan.tools.scm import Version
+from conan.tools.microsoft import is_msvc, unix_path, msvc_runtime_flag
+
 import os
 
-required_conan_version = ">=1.33.0"
+# Temporary duplication until conan 1.54, at which conan.tools.build.stdcpp_library will replace this.
+def _stdcpp_library(conanfile):
+    libcxx = conanfile.settings.get_safe("compiler.libcxx")
+    if libcxx in ["libstdc++", "libstdc++11"]:
+        return "stdc++"
+    elif libcxx in ["libc++"]:
+        return "c++"
+    elif libcxx in ["c++_shared"]:
+        return "c++_shared"
+    elif libcxx in ["c++_static"]:
+        return "c++_static"
+    return None
```

Comment:
```suggestion
```

Please, use https://docs.conan.io/en/latest/reference/tools.html#tools-stdcpp-library instead and add a `TODO` to be updated after 1.54.0. Try avoiding duplicated code from Conan tools.